services/api/handlers/projects.js

```diff
@@ -49,6 +49,25 @@ exports.post = {
 };
 
 exports.get = {
+  oneShallow: function(request, reply) {
+    request.server.methods.projects.findOneShallow(
+      [ request.params.project ],
+      function(err, result) {
+        if (err) {
```

Comment:
@cadecairos @jbuck can't figure out how i'd test this so thinking of removing it